src/test/scala/cornichonTests/CheckoutExistProdTest.scala

```diff
@@ -14,7 +14,6 @@ class CheckoutExistProdTest extends CornichonFeature with FeatureWithProduct {
       And   I createOrderFromCart
       And   I getOrderFromCart
       And   I getCartById
-      Then  assert status.is(200)
```

Comment:
Why have you removed this?

Reply:
there's already an assertion in getCartById `expectedStatus = Some(200)`, this one just duplicates it, doesn't it?

Reply:
You're right!